src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,59 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+
     public void getStatistic(String fromFileName, String toFileName) {
 
+        writeResultFile(toFileName, readAndCalculatedFile(fromFileName));
+    }
+
+    private String readAndCalculatedFile(String fileName) {
+        int[] results = new int[3];
```

Comment:
Using an array to store results is not self-explanatory. Consider using named constants for indices or creating a small class or structure to hold the values with informative names.